src/components/bankaccounts/BankAccountsEditForm.tsx

```diff
@@ -73,34 +75,50 @@ export default function BankAccountsEditForm({ id }: Props) {
         validationSchema={validationSchemaBankAccForm}>
         <Grid container spacing={3}>
           <Grid item xs={12}>
-            <FormTextField type="text" label="bankaccounts:status" name="status" />
+            <FormSelectField
+              label="bankaccounts:fields.status"
+              name="status"
+              options={Object.keys(BankAccountStatus).map((key: string) => {
+                return {
+                  key,
+                  value: BankAccountStatus[key as BankAccountStatus],
+                  name: t(`bankaccounts:status.${key}`),
+                }
+              })}
+            />
           </Grid>
           <Grid item xs={12}>
-            <FormTextField type="text" name="ibanNumber" label="bankaccounts:ibanNumber" />
+            <FormTextField type="text" name="ibanNumber" label="bankaccounts:fields.ibanNumber" />
           </Grid>
           <Grid item xs={12}>
             <FormTextField
               type="text"
               name="accountHolderName"
-              label="bankaccounts:accountHolderName"
+              label="bankaccounts:fields.accountHolderName"
             />
           </Grid>
           <Grid item xs={12}>
-            <FormTextField
-              type="text"
-              name="AccountHolderType"
-              label="bankaccounts:AccountHolderType"
+            <FormSelectField
+              name="accountHolderType"
+              label="bankaccounts:fields.accountHolderType"
+              options={Object.keys(AccountHolderType).map((key: string) => {
+                return {
+                  key,
+                  value: AccountHolderType[key as AccountHolderType],
+                  name: t(`bankaccounts:accountHolderType.${key}`),
+                }
+              })}
             />
           </Grid>
           <Grid item xs={12}>
-            <FormTextField type="text" name="bankName" label="bankaccounts:bankName" />
-          </Grid>
-          <Grid item xs={12}>
-            <FormTextField type="text" name="bankIdCode" label="bankaccounts:bankIdCode" />
+            <FormTextField type="text" name="bankName" label="bankaccounts:fields.bankName" />
           </Grid>
           <Grid item xs={12}>
-            <FormTextField type="text" name="fingerprint" label="bankaccounts:fingerprint" />
+            <FormTextField type="text" name="bankIdCode" label="bankaccounts:fields.bankIdCode" />
           </Grid>
+          {/* <Grid item xs={12}>
+            <FormTextField type="text" name="fingerprint" label="bankaccounts:fields.fingerprint" />
+          </Grid> */}
```

Comment:
Maybe we need to uncomment this

Reply:
Yes, I wasn't sure what to do so I left it commented out.